config/routes.rb

```diff
@@ -1,8 +1,8 @@
 Rails.application.routes.draw do
-  # root 'static_pages/home'
-  get 'static_pages/home'
-  get 'static_pages/help'
-  get 'static_pages/about'
-  get 'static_pages/contact'
-  root 'application#hello'
+  get 'users/new'
+  root 'static_pages#home'
+  get '/help', to: 'static_pages#help'
```

Comment:
`to: 'static_pages#help'` để làm gì e hè

Reply:
khi có request tới /help thì gọi method help ở StaticPagesController
=>> để xác định đúng controller và action